src/assets/data/edition/series/1/section/5/op4/textcritics.json

```diff
@@ -1,5 +1,700 @@
+                            "svgGroupId": "g658",
+                            "measure": "8",
+                            "system": "Ges.",
+                            "position": "9/8",
+                            "comment": "Text sic: Großschreibung <em>Korn</em> (wie <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'text_George_DsR'})\"><strong>George_DsR</strong></a> am Zeilenanfang)."
+                        },
+                        {
+                            "svgGroupId": "g614",
+                            "measure": "10",
+                            "system": "Ges.",
+                            "position": "5.–7. Note",
+                            "comment": "Text sic: Silbentrennung <em>Dä-mmer</em>."
+                        },
+                        {
+                            "svgGroupId": "g615",
+                            "measure": "11",
+                            "system": "Klav. u.",
+                            "position": "1/4",
+                            "comment": "Unterstimmenschicht: Ligaturbogen von T. 10 getilgt. Siehe <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A_corr'})\">Korrekturen in <strong>A</strong></a> T. 10 (8. Note): Bogenanfang rasiert. <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A'})\"><strong>A</strong></a>: Akkoladenwechsel nach T. 10."
+                        },
+                        {
+                            "svgGroupId": "g640",
+                            "measure": "11",
+                            "system": "Ges.",
+                            "position": "6/4",
+                            "comment": "Text sic: Komma nach <em>bannt</em>. <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'text_George_DsR'})\"><strong>George_DsR</strong></a>: kein Komma."
+                        },
+                        {
+                            "svgGroupId": "g287",
+                            "measure": "13",
+                            "system": "Klav. u.",
+                            "position": "(1–4/4)",
+                            "comment": "sic: Unterstimmenschicht mit Ganzer Pause (Vier-Viertel-Pause)."
+                        },
+                        {
+                            "svgGroupId": "g288",
+                            "measure": "14–15",
+                            "system": "Ges.",
+                            "position": "",
+                            "comment": "Vier Ganze Pausen analog Korrektur in <strong>A</strong> T. 14 2. Teiltakt. Siehe <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A_corr'})\">Korrekturen in <strong>A</strong></a>: Zuerst Punktierte Ganze Pause, Viertelpause (T. 14), Halbe Pause, punktierte Halbe Pause, Halbe Pause (T. 15); dann Teiltaktstriche hinzugefügt, Viertelpause T. 14 7/4 gestrichen und Ganze Pause für T. 14 2. Teiltakt hinzugefügt."
```

Comment:
```suggestion
                            "comment": "Vier Ganze Pausen analog Korrektur in <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A'})\"><strong>A</strong></a> T. 14 2. Teiltakt. Siehe <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A_corr'})\">Korrekturen in <strong>A</strong></a>: Zuerst Punktierte Ganze Pause, Viertelpause (T. 14), Halbe Pause, punktierte Halbe Pause, Halbe Pause (T. 15); dann Teiltaktstriche hinzugefügt, Viertelpause T. 14 7/4 gestrichen und Ganze Pause für T. 14 2. Teiltakt hinzugefügt."
```

Reply:
bzw. schon beim ersten Mal auf Korrekturen verlinken? Bzw. beide Teile zusammenfassen:

`Vier Ganze Pausen analog <a (click)=\"ref.navigateToReportFragment({complexId: 'op4', fragmentId: 'source_A_corr'})\">Korrektur in <strong>A</strong></a> T. 14 2. Teiltakt: Zuerst ...` ?

Reply:
Stimmt, ist blöd... - die zusammengefasste Version wäre besser. Möchte aber am Text gegenüber Print nichts ändern. Also lieber Deine suggestion übernehmen!